src/main/java/baritone/utils/schematic/litematica/LitematicaBlockStatePaletteHashMap.java

```diff
@@ -0,0 +1,101 @@
+package baritone.utils.schematic.litematica;
+
+import javax.annotation.Nullable;
+import net.minecraft.block.BlockState;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.nbt.ListNBT;
+import net.minecraft.util.ObjectIntIdentityMap;
+import net.minecraft.nbt.NBTUtil;
+
+public class LitematicaBlockStatePaletteHashMap implements ILitematicaBlockStatePalette
```

Comment:
aNoThEr class taken from the decompiled Minecraft game source. Same course of action should be taken as the others.

Reply:
See the change request for src/main/java/baritone/utils/schematic/litematica/LitematicaBlockStatePaletteLinear.java